backend/types.ts

```diff
@@ -35,3 +37,14 @@ export type NodemailerConfig = {
 };
 
 export type SignUpMethod = "PASSWORD" | "GOOGLE";
+
+export type SchoolTestDTO = {
+  id: string;
+  name: string;
+  duration: number;
+  admin: string;
+  questions: Question[];
+  grade: number;
+};
```

Comment:
I think this can go in the interface (for consistency)

Reply:
nit: add documentation for member variables

Reply:
+1 let's keep this consistent w/ the rest of the codebase and store it in `interfaces`. good catch @mahadzaryab1 !!